### PR TITLE
Visit oromia feature

### PR DESCRIPTION
**Pull request by amanuelbelete95**
**List of Completed Activity**

- Designed a new capstone project
- Followed design guidelines
- Designed the two-page home page and about page
- Designed for Mobile: up to 768px wide.
- Designed for Desktop: 768px or wider
- The home page has a link in the menu to the about page.
- The logo in the header links to the home page.
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- The mobile menu has a close (X) button that closes the menu
- The section "Featured speakers" was created dynamically in JavaScript.
- [Video link](https://www.loom.com/share/376506b30ae048edbf0088e5085f77a0)

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @amanuelbelete95  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that linter checks are not passing. Kindly fix the linter issues before submitting for review.
Screenshot 2023-02-08 at 10 08 49 PM

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -97,36 +86,33 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Website for visitors and tourist**
```
Comment (roshan-bajgain) [inline README.md:89]: - [ ] As per the requirement you should create a video with a maximum length of 5 minutes which contains
`A description of the project.
A demo of the project features (different pages and different screen sizes).
You should also highlight some interesting piece of code or something you built that you are very proud of.`

File: README.md [hunk 2]
```diff
@@ -245,44 +217,26 @@ Feel free to check the [issues page](../../issues/).
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank my coding partners and my coding reviewer for their supporting suggestions.
```
Comment (roshan-bajgain) [inline README.md]: - [ ] As per the requirement your acknowledgments should give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Please include this

File: index.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        </div>
+        <div class="logo-container">
+          <img src="images/bg-visit.jpg" alt="logo" class="logo">
+          <ul class="info-lists">
+              <li><a href="about.html">About</a></li>
+             <li><a href="">Program</a></li>
+             <li><a href="">Join</a></li>
+              <li><a href="">Sponsor</a></li>
+               <li><a href="">News</a></li>
+                <li class="red-border"><a href="">CC Campaign</a></li>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] As per the requirement you should stick to the design as much as possible. The black navbar with a list on the right is missing please implement that and make sure your design matches the original design.
- [ ] If you see the color of the list below that then you will find different colors please use the same color in the list and `CCampaign`
### Your design
his

### Original design
original

File: index.html [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    <div class="about about-page" id="about">
+      
+      <div class="content">
+        <img src="images/bale park.jpg" alt="" class="mobile-image"/>
+        <h2 class="content-heading">Balee Park of Oromia</h2>
+        <p class="place-info">
+          is located 400 km southeast of Addis Ababa in Oromia National Regional
+          State in south-eastern Ethiopia.
+        </p>
+      </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you please increase the font weight of the heading. Also if you see the content after that heading then you will find the border with min-width please implement accordingly.
### Your design
hiz

### Original design
original

File: index.html [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+          the Indian Ocean.
+        </p>
+      </div>
+
+
+    </div>
+          <div class="button-container">
+       <button class="see-more">More About Oromia</button> 
+      </div>
+
```
Comment (roshan-bajgain) [inline index.html]: - [ ] You should stick to the design as much as possible. please use the border on every card and make the size adjustable as there is a gap between the left and right and `Historic places in Oromia to Visit` is inside the black background. See the image below for reference.
- [ ] Also make sure you have black stripes background. See image below for reference
### Your design
his

### Original design
fimga

File: index.html [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+    <div class="heading-hr">
+        <h1 class="about-heading">Featured Speakers</h1>
+        <hr class="hr" />
+      </div>
+
+    <div class="feature-container" id="speaker">
+      
+    </div>
+
+    <div class="footer" id="">
```
Comment (roshan-bajgain) [inline index.html]: - [ ] Please adjust the spacing to match the original design. The first speaker and the second should be on the same line. The image covers the equal height with the content on the right of the image. Please try to adjust the spacing. See the image below for reference. Also please decrease the number of height and make is smaller to match the design
### Your design
his

### Figma design

original

File: about.html [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+          resources and cultural heritages. Visit Oromia where the sheer
+          diversity of the culture, people and landscape is sure to leave you
+          astounded.
+        </p>
+
+        <div class="contact-us">
+          <p class="info">
+            Please contact-us for further explanation about visit oromia @
+            <br /><span class="creative-heading">VisitOromia@2023</span>
+          </p>
```
Comment (roshan-bajgain) [inline about.html]: - [ ] In this about section, the page title at the middle of the page, and the content should have a border with different background. This `VisitOromia@2023` should be central to the page please implement it.
- [ ] If you see the design then you will find a white background. See implement it. Please remove the `"Hello, Tourist!"` in the about section.
### Your design
his

### Original design
orignal

File: about.html [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+        </p>
+      </div>
+    <div class="logo-about logo-container">
+      <img src="images/bg-visit.jpg" alt="" />
+    </div>
+    <div class="live-oromia">
+      <div class="see-people">
+        <h2 class="creative-heading">
+          See Previous Visit Oromia Gallery in 2010 and 2016
+        </h2>
```
Comment (roshan-bajgain) [inline about.html]: - [ ] If you see the design then the title is in the middle and the content should have limited width please do so. There is gap on left and right
- [ ] There is a `hx` tag after the logo and the logo has border on it please see the design and implement it. Please use the same color and font for content
### Your design
his

### Original design
original

File: about.html [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+        <img src="images/awash.jpg" alt="" />
+        <img src="images/ethiopian-air.png" />
+        <img src="images/land eth.png" alt="" />
+        <img src="images/oda.jpg" alt="" />
+      </div>
+    </div>
+    <div class="creative-common" id="creative">
+      <img src="images/bg-visit.jpg" alt="" />
+      <h3>@2015 Creative Common, Oromia, Ethiopia</h3>
+    </div>
```
Comment (roshan-bajgain) [inline about.html]: - [ ] As per the requirement the image should have gaps on right and left of the page and the footer has a black background and content on the right side of the footer. Please implement it. See the image below for reference
### Your design
his


### Figma design

original

File: script.js [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+  menu.classList.toggle('hidden-menu');
+});
+
+function menuRemover() {
+  menu.classList.toggle('hidden-menu');
+}
+
+menuItems.forEach((item) => {
+  item.addEventListener('click', menuRemover);
+});
```
Comment (roshan-bajgain) [inline script.js:18]: - [ ] When the hamburger buttons are open in the mobile version then I cannot go to the about section by clicking on `about section please fix it. There should be both menus for `Home Page` and `About Page` for navigation. Please do so as well.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+            <h2 class="interval">2022.10.17(MON) ~ 30(SUN)</h2>
+            <p class="place">@Addis Ababa, Ethiopia and more</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    
+    <div class="heading-hr" id="about">
+        <h1 class="about-heading">Historic places in Oromia to Visit</h1>
+        <hr class="hr" />
```
Comment (roshan-bajgain) [inline index.html]: - [ ] There should not be more than 1 `h1` tag per page please remove the other `h1` tag here for semantic reason

File: index.html [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+      </div>
+
+
+    </div>
+          <div class="button-container">
+       <button class="see-more">More About Oromia</button> 
+      </div>
+
+    <div class="heading-hr">
+        <h1 class="about-heading">Featured Speakers</h1>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] There should not be more than 1 `h1` tag per page please remove the other `h1` tag here for semantic reason

File: index.html [hunk 13]
```diff
@@ -0,0 +1,166 @@
...
+  <body>
+    <div class="menu-bar hidden-menu">
+      <button class="close-btn">
+              <i class="fa fa-times"></i>
+      </button>
+      <div class="menu-mobile">
+           <a href="#about">About</a>
+            <a href="#program">Program</a>
+            <a href="#speaker">Speaker</a>
+      </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] Please wrap this section with `nav` tag for better semantic HTML.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,166 @@
...
+
+    <div class="heading-hr">
+        <h1 class="about-heading">Featured Speakers</h1>
+        <hr class="hr" />
+      </div>
+
+    <div class="feature-container" id="speaker">
+      
+    </div>
+
```
Comment (roshan-bajgain) [inline index.html]: - [ ] It's better to use section tag here for better semantic HTML.

File: index.html [hunk 15]
```diff
@@ -0,0 +1,166 @@
...
+      </div>
+
+      <div class="creative-common" id="creative">
+
+        <img src="images/bg-visit.jpg" alt="">
+        <h3>@2015 Creative Common, Oromia, Ethiopia</h3>
+        
+
+      </div>
+    </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] It's better to use a footer tag in the footer for better semantic HTML.

File: about.html (hunk 9 above)
Comment (roshan-bajgain) [inline about.html]: - [ ] Please use section tag for better semantic HTML.

Comment (roshan-bajgain) [inline about.html]: - [ ] I would suggest you please use the footer tag for the footer it indicates better semantic HTML.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @amanuelbelete95, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 16]
```diff
@@ -0,0 +1,166 @@
...
+        </div>
+        <div class="logo-container">
+          <img src="images/bg-visit.jpg" alt="logo" class="logo">
+          <ul class="info-lists">
+              <li><a href="about.html">About</a></li>
+             <li><a href="">Program</a></li>
+             <li><a href="">Join</a></li>
+              <li><a href="">Sponsor</a></li>
+               <li><a href="">News</a></li>
+                <li class="red-border"><a href="">CC Campaign</a></li>
```
Comment (osamaashraf6) [inline index.html]: - [ ] As stated by the previous reviewer kindly try to stick to the color of the text here to be nice like the Behance 😊
- [ ] Also please give the cc campaign the correct color like the Behance 💯

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  image    |    image   |

File: index.html [hunk 17]
```diff
@@ -0,0 +1,166 @@
...
+          <h4 class="heading2">"Hello, Tourist!"</h4>
+          <h1 class="home-heading">
+            Visit Oromia, Ethiopia <br> Land of Origin and Heritages in 2023!
+          </h1>
+          <p class="info info-main">
+            Oromia is a largest region in Ethiopia with potential natural
+            resources and cultural heritages. Visit Oromia where the sheer
+            diversity of the culture, people and landscape is sure to leave you
+            astounded.
+          </p>
```
Comment (osamaashraf6) [inline index.html:70]: - [ ] You did a great job at adding the border at the content, but as stated by the previous reviewer kindly try to give min width to the content to be like the Behance

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|   image   |     image  |

File: index.html [hunk 18]
```diff
@@ -0,0 +1,166 @@
...
+      </div>
+    <section class="about about-page" id="about">
+      
+      <div class="content">
+        <img src="images/bale park.jpg" alt="" class="mobile-image"/>
+        <h2 class="content-heading">Balee Park of Oromia</h2>
+        <p class="place-info">
+          is located 400 km southeast of Addis Ababa in Oromia National Regional
+          State in south-eastern Ethiopia.
+        </p>
```
Comment (osamaashraf6) [inline index.html]: - [ ] You did well at adding the border, but  As stated by the previous reviewer  kindly  try to make the size adjustable as there is a gap between the left and right and Historic places in Oromia to Visit are inside the black background. See the image below for reference.
- [ ] Also try to give the section the same background image like the Behance to be nice 

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|    image  |    image   |

File: script.js [hunk 19]
```diff
@@ -0,0 +1,82 @@
...
+},
+
+{
+  name: 'Denzel Washington',
+  speakerImg: 'images/denzel.jpg',
+  profession: 'An American actor and filmmaker',
+  statement: "When I visited Bale National Park of Oromia, I couldn't believe that these amazing natural beauty existed in the region. Different animals, birds and trees took my eyes.",
+
+},
+
```
Comment (osamaashraf6) [inline script.js:36]: - [ ] As stated by the previous reviewer here Please adjust the spacing to match the original design. The first speaker and the second should be on the same line. The image covers the equal height with the content on the right of the image. Please try to adjust the spacing. See the image below for reference. Also please decrease the height and make is smaller to match the design💯

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  image    |   image    |

File: about.html [hunk 20]
```diff
@@ -0,0 +1,118 @@
...
+      <div class="description">
+        <h4 class="heading2">"Hello, Tourist!"</h4>
+        <h1 class="home-heading">
+          Visit Oromia, Ethiopia Land of Origin and Heritages in 2023!
+        </h1>
+        <p class="info info-about">
+          Oromia is a largest region in Ethiopia with potential natural
+          resources and cultural heritages. Visit Oromia where the sheer
+          diversity of the culture, people and landscape is sure to leave you
+          astounded.
```
Comment (osamaashraf6) [inline about.html]: - [ ] As stated by the previous reviewer kindly try In this about section, to make the page title at the middle of the page  This VisitOromia@2023 should be central to the page please implement it.
- [ ] Also If you see the design then you will find a white background. See implement it. Please remove the "Hello, Tourist!" in the about section.

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|    image  |   image   |

File: about.html [hunk 21]
```diff
@@ -0,0 +1,118 @@
...
+      </div>
+    </section>
+    <section class="creative-container">
+        <h2 class="creative-heading">
+          Creative Commons Visit Oromia Logo in 2023!
+        </h2>
+        <hr />
+        <p class="about-info info">
+          The logo of visit Oromia was developed by the group of software
+          developer  who were qualified by visit-oromia tourism commission. <br> After
```
Comment (osamaashraf6) [inline about.html:81]: - [ ] Kindly as stated by the previous reviewer If you see the design then the title is in the middle and the content should have limited width please do so. There is a gap between left and right

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  image    |     image  |

File: about.html [hunk 22]
```diff
@@ -0,0 +1,118 @@
...
+          <h2>2010</h2>
+          <p>Visit Oromia in 2010</p>
+        </div>
+      </div>
+    </section>
+    <section class="hidden-partner partner" id="partner">
+      <div class="partner-hr">
+        <h2>Partner</h2>
+        <hr class="part" />
+      </div>
```
Comment (osamaashraf6) [inline about.html:113]: - [ ] As stated by the previous reviewer As per the requirement the image should have gaps on right and left of the page and the footer has a black background and content on the right side of the footer. Please implement it. See the image below for reference

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  image    |    image  |

File: script.js (hunk 10 above)
Comment (osamaashraf6) [inline script.js:18]: - [ ] As stated here by the previous reviewer  When the hamburger buttons are open in the mobile version then I cannot go to the about section by clicking on the about section please fix it. There should be both menus for Home Page and About Page for navigation. Please do so as well.
- [ ] Also [Optional]  kindly consider fixing the responsiveness at the mobile version as you see there are some issues in the responsiveness😊

> Check below to see the difference 😊⤵️

| Your design| 
| --- | 
|   Recording 2023-02-09 at 17 45 56  |

File: index.html [hunk 23]
```diff
@@ -0,0 +1,166 @@
...
+            <h2 class="interval">2022.10.17(MON) ~ 30(SUN)</h2>
+            <p class="place">@Addis Ababa, Ethiopia and more</p>
+          </div>
+        </div>
+      </section>
+    </div>
+    
+    <div class="heading-hr" id="about">
+        <h1 class="about-heading">Historic places in Oromia to Visit</h1>
+        <hr class="hr" />
```
Comment (osamaashraf6) [inline index.html]: - [ ] As stated here by the previous reviewer please, There should not be more than 1 h1 tag per page please remove the other h1 tag here for a semantic reasons😊

File: about.html [hunk 24]
```diff
@@ -0,0 +1,118 @@
...
+        </div>
+ </section>
+
+
+    <section class="home-page">
+      <div class="description">
+        <h4 class="heading2">"Hello, Tourist!"</h4>
+        <h1 class="home-heading">
+          Visit Oromia, Ethiopia Land of Origin and Heritages in 2023!
+        </h1>
```
Comment (osamaashraf6) [inline about.html]: - [ ]  As stated here by the previous reviewer please, There should not be more than 1 h1 tag per page please remove the other h1 tag here for a semantic reasons😊

File: about.html [hunk 25]
```diff
@@ -0,0 +1,118 @@
...
+        <div class="live-container2">
+          <h2>2010</h2>
+          <p>Visit Oromia in 2010</p>
+        </div>
+      </div>
+    </section>
+    <section class="hidden-partner partner" id="partner">
+      <div class="partner-hr">
+        <h2>Partner</h2>
+        <hr class="part" />
```
Comment (osamaashraf6) [inline about.html:112]: - [ ] As stated by the previous reviewer I would suggest you please use the footer tag for the footer it indicates better semantic HTML.😊

File: README.md [hunk 26]
```diff
@@ -245,44 +219,27 @@ Feel free to check the [issues page](../../issues/).
...
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank Cindy Shin for the design template provided.
+Originally designed by: [Cindy shin]{https://www.behance.net/adagio07}
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (osamaashraf6) [inline README.md]: - [ ] You did well at writing about the original design,but I see that the link for the design is broken,so please try to use the correct link as stated by the previous reviewer 😊

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @amanuelbelete95 😊, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
you-are-near


## Highlights💯🎈🪄✔️

- [x] Descriptive READme file in a professional way.✔️
- [x] No linter errors. ✔️
- [x] Good usage in correcting GitHub flow.✔️
- [x] Nice design.✔️


## Suggested changes
- You did well by recording the loom video, but as I see you didn't show us The video contains a demo of the project, showing the pages in different screen sizes, so kindly try to make that 😊for the requirements 

- Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: about.html [hunk 27]
```diff
@@ -0,0 +1,129 @@
...
+             <li><a href="">Join</a></li>
+              <li><a href="">Sponsor</a></li>
+               <li><a href="">News</a></li>
+                <li class="red-border"><a href="">CC Campaign</a></li>
+            </ul>
+        </div>
+ </section>
+
+
+    <section class="home-page">
```
Comment (Kamba56) [inline about.html:53]: - [ ] As stated by the previous reviewers, kindly centralize the contents of this section to match the given template.
### Your design
image
### Design template
f1

File: about.html [hunk 28]
```diff
@@ -0,0 +1,129 @@
...
+    </div>
+    <section class="live-oromia" id="live">
+      <div class="see-people">
+        <h2 class="creative-heading">
+          See Previous Visit Oromia Gallery in 2010 and 2016
+        </h2>
+        <p>Look at the people from Land of Oromia, Ethiopia in the pasts</p>
+      </div>
+
+      <div class="live-cont">
```
Comment (Kamba56) [inline about.html:98]: - [ ] As requested by previous reviewers, kindly add spaces to the left and right of this container in the desktop view. Consider adding a padding left and right of 200px to the destop view to match the design template.
### Your design
image

### Design template
f2

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @amanuelbelete95, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by victorokoroji — APPROVED** [review]
Hi @amanuelbelete95, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**